src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -248,11 +249,11 @@ const convertFrameworkDetail = async (framework) => {
       retryDelayTime: framework.status.retryPolicyStatus.retryDelaySec,
       createdTime: new Date(framework.metadata.creationTimestamp).getTime(),
       completedTime: new Date(framework.status.completionTime).getTime(),
-      appId: framework.status.attemptStatus.instanceUID,
+      appId: attemptStatus.instanceUID,
       appProgress: completionStatus ? 1 : 0,
       appTrackingUrl: '',
-      appLaunchedTime: new Date(framework.metadata.creationTimestamp).getTime(),
-      appCompletedTime: new Date(framework.status.completionTime).getTime(),
+      appLaunchedTime: new Date(attemptStatus.runTime || attemptStatus.completionTime).getTime(),
```

Comment:
Also update related doc for appLaunchedTime.

For a job attempt:
WholeDuration = CompletionTime - StartTime
RunningDuration = CompletionTime - LaunchedTime
WaitingDuration = WholeDuration - RunningDuration

For the whole job which may have multiple attempts:
WholeDuration = CompletionTime - StartTime
RunningTime = Sum_all_attempts (CompletionTime - LaunchedTime)
WaitingDuration = WholeDuration - RunningDuration

Reply:
added